python/daqconf/core/conf_utils.py

```diff
@@ -777,3 +782,45 @@ def write_json_files(app_command_datas, system_command_datas, json_dir, verbose=
             json.dump(cfg, f, indent=4, sort_keys=True)
 
     console.log(f"System configuration generated in directory '{json_dir}'")
+
+
+def get_version():
+    from os import getenv
+    version = getenv("DUNE_DAQ_BASE_RELEASE")
+    if not version:
+        raise RuntimeError('Utils: dunedaq version not in the variable env DUNE_DAQ_BASE_RELEASE! Exit nanorc and\nexport DUNE_DAQ_BASE_RELEASE=dunedaq-vX.XX.XX\n')
+    return version
+
+def nightly_or_release(version):
+    from re import match
+    if   match( 'N[0-9]{2}-[0-9]{2}-[0-9]{2}', version): return 'nightly'
+    elif match('NT[0-9]{2}-[0-9]{2}-[0-9]{2}', version): return 'nightly'
```

Comment:
I think the letter is arbitrary...maybe `N[A-Z]?[0-9]` would be better

Reply:
Also candidate releases have a different naming pattern.

Reply:
Ok, I've done this even better now.